cmd/rollkit/commands/intercept_test.go

```diff
@@ -0,0 +1,103 @@
+package commands
+
+import (
+	"errors"
+	"os"
+	"testing"
+
+	rollconf "github.com/rollkit/rollkit/config"
+
+	"github.com/spf13/cobra"
+)
+
+func TestInterceptCommand(t *testing.T) {
+	tests := []struct {
+		name              string
+		rollkitCommands   []string
+		mockReadToml      func() (rollconf.TomlConfig, error)
+		mockRunEntrypoint func(rollkitConfig *rollconf.TomlConfig, args []string) error
+		args              []string
+		wantErr           bool
+		wantExecuted      bool
+	}{
+		{
+			name: "Successful intercept with entrypoint",
+			mockReadToml: func() (rollconf.TomlConfig, error) {
+				return rollconf.TomlConfig{
+					Entrypoint: "test-entrypoint",
+					Chain:      rollconf.ChainTomlConfig{ConfigDir: "/test/config"},
+
+					RootDir: "/test/root",
+				}, nil
+			},
+			mockRunEntrypoint: func(config *rollconf.TomlConfig, flags []string) error {
+				return nil
+			},
+			args:         []string{"cmd", "arg1", "arg2"},
+			wantErr:      false,
+			wantExecuted: true,
+		},
+		{
+			name: "Configuration read error",
+			mockReadToml: func() (rollconf.TomlConfig, error) {
+				return rollconf.TomlConfig{}, errors.New("read error")
+			},
+			args:         []string{"cmd"},
+			wantErr:      true,
+			wantExecuted: false,
+		},
+		{
+			name: "Empty entrypoint",
+			mockReadToml: func() (rollconf.TomlConfig, error) {
+				return rollconf.TomlConfig{Entrypoint: ""}, nil
+			},
+			args:         []string{"cmd"},
+			wantErr:      true,
+			wantExecuted: true,
+		},
+		{
+			name:            "Skip intercept",
+			rollkitCommands: []string{"cmd1, cmd2"},
+			mockReadToml: func() (rollconf.TomlConfig, error) {
+				return rollconf.TomlConfig{
+					Entrypoint: "test-entrypoint",
+					Chain:      rollconf.ChainTomlConfig{ConfigDir: "/test/config"},
+
+					RootDir: "/test/root",
+				}, nil
+			},
+			mockRunEntrypoint: func(config *rollconf.TomlConfig, flags []string) error {
+				return nil
+			},
+			args:         []string{"cmd"},
+			wantErr:      false,
+			wantExecuted: true,
```

Comment:
This test case seems wrong. Shouldn't skipping the intercept mean no toml and no entrypoint with `wantErr` and `wantExecuted` to be false?